2025-04-22 14:44:36,556 INFO    StreamThr :578817 [internal.py:wandb_internal():85] W&B internal server running at pid: 578817, started at: 2025-04-22 14:44:36.555697
2025-04-22 14:44:36,557 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status
2025-04-22 14:44:36,559 INFO    WriterThread:578817 [datastore.py:open_for_write():87] open: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/run-6wyz8vcn.wandb
2025-04-22 14:44:36,560 DEBUG   SenderThread:578817 [sender.py:send():379] send: header
2025-04-22 14:44:36,560 DEBUG   SenderThread:578817 [sender.py:send():379] send: run
2025-04-22 14:44:38,044 INFO    SenderThread:578817 [dir_watcher.py:__init__():211] watching files in: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files
2025-04-22 14:44:38,044 INFO    SenderThread:578817 [sender.py:_start_run_threads():1188] run started: 6wyz8vcn with start time 1745304276.555625
2025-04-22 14:44:38,056 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: check_version
2025-04-22 14:44:38,056 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: check_version
2025-04-22 14:44:40,053 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: run_start
2025-04-22 14:44:40,080 DEBUG   HandlerThread:578817 [system_info.py:__init__():26] System info init
2025-04-22 14:44:40,080 DEBUG   HandlerThread:578817 [system_info.py:__init__():41] System info init done
2025-04-22 14:44:40,080 INFO    HandlerThread:578817 [system_monitor.py:start():194] Starting system monitor
2025-04-22 14:44:40,080 INFO    SystemMonitor:578817 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-04-22 14:44:40,080 INFO    HandlerThread:578817 [system_monitor.py:probe():214] Collecting system info
2025-04-22 14:44:40,080 INFO    SystemMonitor:578817 [interfaces.py:start():188] Started cpu monitoring
2025-04-22 14:44:40,083 INFO    SystemMonitor:578817 [interfaces.py:start():188] Started disk monitoring
2025-04-22 14:44:40,084 INFO    SystemMonitor:578817 [interfaces.py:start():188] Started gpu monitoring
2025-04-22 14:44:40,084 INFO    SystemMonitor:578817 [interfaces.py:start():188] Started memory monitoring
2025-04-22 14:44:40,086 INFO    SystemMonitor:578817 [interfaces.py:start():188] Started network monitoring
2025-04-22 14:44:40,120 DEBUG   HandlerThread:578817 [system_info.py:probe():152] Probing system
2025-04-22 14:44:40,130 DEBUG   HandlerThread:578817 [gitlib.py:_init_repo():56] git repository is invalid
2025-04-22 14:44:40,130 DEBUG   HandlerThread:578817 [system_info.py:probe():200] Probing system done
2025-04-22 14:44:40,130 DEBUG   HandlerThread:578817 [system_monitor.py:probe():223] {'os': 'Linux-5.15.0-134-generic-x86_64-with-glibc2.31', 'python': '3.9.21', 'heartbeatAt': '2025-04-22T06:44:40.120801', 'startedAt': '2025-04-22T06:44:36.550417', 'docker': None, 'cuda': None, 'args': ('task=mw-reach', 'steps=10000'), 'state': 'running', 'program': '/home/levi/Documents/MOORE-TDMPC/moore_tdmpc/train_moore.py', 'codePathLocal': None, 'host': 'workstation', 'username': 'levi', 'executable': '/home/levi/anaconda3/envs/tdmpc2/bin/python', 'cpu_count': 16, 'cpu_count_logical': 24, 'cpu_freq': {'current': 3303.1405833333333, 'min': 800.0, 'max': 4950.0}, 'cpu_freq_per_core': [{'current': 1075.374, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5400.0}, {'current': 3400.0, 'min': 800.0, 'max': 5400.0}, {'current': 3400.0, 'min': 800.0, 'max': 5400.0}, {'current': 3400.0, 'min': 800.0, 'max': 5400.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 5300.0}, {'current': 3400.0, 'min': 800.0, 'max': 4200.0}, {'current': 3400.0, 'min': 800.0, 'max': 4200.0}, {'current': 3400.0, 'min': 800.0, 'max': 4200.0}, {'current': 3400.0, 'min': 800.0, 'max': 4200.0}, {'current': 3400.0, 'min': 800.0, 'max': 4200.0}, {'current': 3400.0, 'min': 800.0, 'max': 4200.0}, {'current': 3400.0, 'min': 800.0, 'max': 4200.0}, {'current': 3400.0, 'min': 800.0, 'max': 4200.0}], 'disk': {'/': {'total': 456.8756866455078, 'used': 392.09399032592773}}, 'gpu': 'NVIDIA GeForce RTX 4090 D', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090 D', 'memory_total': 25757220864}], 'memory': {'total': 62.60572814941406}}
2025-04-22 14:44:40,130 INFO    HandlerThread:578817 [system_monitor.py:probe():224] Finished collecting system info
2025-04-22 14:44:40,130 INFO    HandlerThread:578817 [system_monitor.py:probe():227] Publishing system info
2025-04-22 14:44:40,131 DEBUG   HandlerThread:578817 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2025-04-22 14:44:41,000 DEBUG   HandlerThread:578817 [system_info.py:_save_conda():224] Saving conda packages done
2025-04-22 14:44:41,001 INFO    HandlerThread:578817 [system_monitor.py:probe():229] Finished publishing system info
2025-04-22 14:44:41,004 DEBUG   SenderThread:578817 [sender.py:send():379] send: files
2025-04-22 14:44:41,004 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2025-04-22 14:44:41,046 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: python_packages
2025-04-22 14:44:41,046 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: python_packages
2025-04-22 14:44:41,046 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: stop_status
2025-04-22 14:44:41,047 INFO    Thread-12 :578817 [dir_watcher.py:_on_file_created():271] file/dir created: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/conda-environment.yaml
2025-04-22 14:44:41,047 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: stop_status
2025-04-22 14:44:41,047 INFO    Thread-12 :578817 [dir_watcher.py:_on_file_created():271] file/dir created: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/wandb-metadata.json
2025-04-22 14:44:41,048 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 14:44:42,047 INFO    Thread-12 :578817 [dir_watcher.py:_on_file_created():271] file/dir created: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/requirements.txt
2025-04-22 14:44:42,047 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 14:44:42,435 DEBUG   SenderThread:578817 [sender.py:send():379] send: telemetry
2025-04-22 14:44:42,436 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:44:42,551 DEBUG   SenderThread:578817 [sender.py:send():379] send: telemetry
2025-04-22 14:44:42,591 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,591 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,592 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,592 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,593 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:42,644 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,645 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,645 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,645 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:42,682 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,682 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,682 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,683 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:42,719 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,720 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,720 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,720 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:42,756 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,756 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,756 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,756 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:42,792 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,792 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,793 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,793 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:42,829 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,829 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,829 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,829 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:42,865 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,866 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,866 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,866 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:42,902 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,902 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,902 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,902 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:42,942 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: partial_history
2025-04-22 14:44:42,942 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:42,942 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:42,942 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:43,047 INFO    Thread-12 :578817 [dir_watcher.py:_on_file_created():271] file/dir created: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/output.log
2025-04-22 14:44:43,047 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 14:44:43,047 INFO    Thread-12 :578817 [dir_watcher.py:_on_file_created():271] file/dir created: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/wandb-summary.json
2025-04-22 14:44:43,201 DEBUG   SenderThread:578817 [sender.py:send():379] send: exit
2025-04-22 14:44:43,201 INFO    SenderThread:578817 [sender.py:send_exit():586] handling exit code: 1
2025-04-22 14:44:43,201 INFO    SenderThread:578817 [sender.py:send_exit():588] handling runtime: 3
2025-04-22 14:44:43,201 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:43,201 INFO    SenderThread:578817 [sender.py:send_exit():594] send defer
2025-04-22 14:44:43,201 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:43,202 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 0
2025-04-22 14:44:43,202 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:43,202 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 0
2025-04-22 14:44:43,202 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 1
2025-04-22 14:44:43,202 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:43,202 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 1
2025-04-22 14:44:43,202 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:43,202 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 1
2025-04-22 14:44:43,202 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 2
2025-04-22 14:44:43,202 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:43,202 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 2
2025-04-22 14:44:43,202 INFO    HandlerThread:578817 [system_monitor.py:finish():203] Stopping system monitor
2025-04-22 14:44:43,202 INFO    HandlerThread:578817 [interfaces.py:finish():200] Joined cpu monitor
2025-04-22 14:44:43,202 DEBUG   SystemMonitor:578817 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-04-22 14:44:43,202 INFO    HandlerThread:578817 [interfaces.py:finish():200] Joined disk monitor
2025-04-22 14:44:43,202 DEBUG   SystemMonitor:578817 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2025-04-22 14:44:43,203 DEBUG   SystemMonitor:578817 [system_monitor.py:_start():183] Publishing last batch of metrics
2025-04-22 14:44:43,229 INFO    HandlerThread:578817 [interfaces.py:finish():200] Joined gpu monitor
2025-04-22 14:44:43,229 INFO    HandlerThread:578817 [interfaces.py:finish():200] Joined memory monitor
2025-04-22 14:44:43,229 INFO    HandlerThread:578817 [interfaces.py:finish():200] Joined network monitor
2025-04-22 14:44:43,229 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:43,229 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 2
2025-04-22 14:44:43,229 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 3
2025-04-22 14:44:43,229 DEBUG   SenderThread:578817 [sender.py:send():379] send: stats
2025-04-22 14:44:43,229 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:43,229 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 3
2025-04-22 14:44:43,230 DEBUG   SenderThread:578817 [sender.py:send():379] send: history
2025-04-22 14:44:43,230 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: summary_record
2025-04-22 14:44:43,230 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:43,230 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:43,230 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 3
2025-04-22 14:44:43,230 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 4
2025-04-22 14:44:43,230 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:43,230 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 4
2025-04-22 14:44:43,230 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:43,230 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 4
2025-04-22 14:44:43,230 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 5
2025-04-22 14:44:43,230 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:43,230 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 5
2025-04-22 14:44:43,230 DEBUG   SenderThread:578817 [sender.py:send():379] send: summary
2025-04-22 14:44:43,230 INFO    SenderThread:578817 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-04-22 14:44:43,231 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:43,231 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 5
2025-04-22 14:44:43,231 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 6
2025-04-22 14:44:43,231 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:43,231 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 6
2025-04-22 14:44:43,231 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:43,231 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 6
2025-04-22 14:44:43,232 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:44:43,593 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 7
2025-04-22 14:44:43,593 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:43,593 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 7
2025-04-22 14:44:43,593 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:43,593 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 7
2025-04-22 14:44:44,047 INFO    Thread-12 :578817 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/config.yaml
2025-04-22 14:44:44,048 INFO    Thread-12 :578817 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/wandb-summary.json
2025-04-22 14:44:44,200 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:45,048 INFO    Thread-12 :578817 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/output.log
2025-04-22 14:44:45,164 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 8
2025-04-22 14:44:45,164 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:45,165 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:45,165 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 8
2025-04-22 14:44:45,165 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:45,165 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 8
2025-04-22 14:44:45,165 INFO    SenderThread:578817 [job_builder.py:build():440] Attempting to build job artifact
2025-04-22 14:44:45,165 INFO    SenderThread:578817 [job_builder.py:_get_source_type():580] no source found
2025-04-22 14:44:45,165 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 9
2025-04-22 14:44:45,165 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:45,165 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 9
2025-04-22 14:44:45,165 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:45,165 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 9
2025-04-22 14:44:45,166 INFO    SenderThread:578817 [dir_watcher.py:finish():358] shutting down directory watcher
2025-04-22 14:44:45,200 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:45,437 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:45,437 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3603'}
2025-04-22 14:44:45,437 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:46,048 INFO    Thread-12 :578817 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/output.log
2025-04-22 14:44:46,049 INFO    SenderThread:578817 [dir_watcher.py:finish():388] scan: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files
2025-04-22 14:44:46,050 INFO    SenderThread:578817 [dir_watcher.py:finish():402] scan save: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/conda-environment.yaml conda-environment.yaml
2025-04-22 14:44:46,050 INFO    SenderThread:578817 [dir_watcher.py:finish():402] scan save: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/wandb-summary.json wandb-summary.json
2025-04-22 14:44:46,050 INFO    SenderThread:578817 [dir_watcher.py:finish():402] scan save: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/output.log output.log
2025-04-22 14:44:46,050 INFO    SenderThread:578817 [dir_watcher.py:finish():402] scan save: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/config.yaml config.yaml
2025-04-22 14:44:46,050 INFO    SenderThread:578817 [dir_watcher.py:finish():402] scan save: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/wandb-metadata.json wandb-metadata.json
2025-04-22 14:44:46,051 INFO    SenderThread:578817 [dir_watcher.py:finish():402] scan save: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/files/requirements.txt requirements.txt
2025-04-22 14:44:46,055 INFO    SenderThread:578817 [sender.py:transition_state():614] send defer: 10
2025-04-22 14:44:46,056 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:46,056 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: defer
2025-04-22 14:44:46,056 INFO    HandlerThread:578817 [handler.py:handle_request_defer():184] handle defer: 10
2025-04-22 14:44:46,065 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: defer
2025-04-22 14:44:46,069 INFO    SenderThread:578817 [sender.py:send_request_defer():610] handle sender defer: 10
2025-04-22 14:44:46,074 INFO    SenderThread:578817 [file_pusher.py:finish():169] shutting down file pusher
2025-04-22 14:44:46,202 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:46,202 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:46,835 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:46,835 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '6174'}
2025-04-22 14:44:46,835 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:46,844 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:46,844 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '403'}
2025-04-22 14:44:46,844 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:47,202 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:47,204 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:47,527 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:47,527 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2419'}
2025-04-22 14:44:47,527 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:47,530 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:47,530 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3603'}
2025-04-22 14:44:47,530 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:47,677 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:47,677 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3795'}
2025-04-22 14:44:47,677 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:48,205 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:48,206 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:48,721 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:48,722 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '403'}
2025-04-22 14:44:48,722 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:49,112 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:49,112 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2419'}
2025-04-22 14:44:49,113 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:49,113 INFO    wandb-upload_5:578817 [retry.py:__call__():172] Retry attempt failed:
urllib3.exceptions.SSLError: TLS/SSL connection has been closed (EOF) (_ssl.c:1147)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/urllib3/connectionpool.py", line 841, in urlopen
    retries = retries.increment(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/urllib3/util/retry.py", line 519, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2880, in upload_file
    response = self._upload_file_session.put(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 698, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2911, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2880, in upload_file
    response = self._upload_file_session.put(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 698, in send
    raise SSLError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2025-04-22 14:44:49,163 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:49,163 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3795'}
2025-04-22 14:44:49,163 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:49,163 INFO    wandb-upload_4:578817 [retry.py:__call__():172] Retry attempt failed:
urllib3.exceptions.SSLError: TLS/SSL connection has been closed (EOF) (_ssl.c:1147)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/urllib3/connectionpool.py", line 841, in urlopen
    retries = retries.increment(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/urllib3/util/retry.py", line 519, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2880, in upload_file
    response = self._upload_file_session.put(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 698, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2911, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2880, in upload_file
    response = self._upload_file_session.put(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 698, in send
    raise SSLError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2025-04-22 14:44:49,206 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:49,207 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:49,351 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:49,351 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '6174'}
2025-04-22 14:44:49,351 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:49,351 INFO    wandb-upload_1:578817 [retry.py:__call__():172] Retry attempt failed:
urllib3.exceptions.SSLError: TLS/SSL connection has been closed (EOF) (_ssl.c:1147)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/urllib3/connectionpool.py", line 841, in urlopen
    retries = retries.increment(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/urllib3/util/retry.py", line 519, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2880, in upload_file
    response = self._upload_file_session.put(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 698, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2911, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2880, in upload_file
    response = self._upload_file_session.put(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 698, in send
    raise SSLError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2025-04-22 14:44:49,483 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:49,484 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8327'}
2025-04-22 14:44:49,484 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:49,484 INFO    wandb-upload_3:578817 [retry.py:__call__():172] Retry attempt failed:
urllib3.exceptions.SSLError: TLS/SSL connection has been closed (EOF) (_ssl.c:1147)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/urllib3/connectionpool.py", line 841, in urlopen
    retries = retries.increment(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/urllib3/util/retry.py", line 519, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2880, in upload_file
    response = self._upload_file_session.put(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 698, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2911, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2880, in upload_file
    response = self._upload_file_session.put(
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 649, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/levi/anaconda3/envs/tdmpc2/lib/python3.9/site-packages/requests/adapters.py", line 698, in send
    raise SSLError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2025-04-22 14:44:50,208 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:50,209 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:50,421 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:50,422 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3603'}
2025-04-22 14:44:50,422 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:51,209 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:51,210 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:51,311 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:51,311 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8327'}
2025-04-22 14:44:51,311 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:51,348 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:51,349 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '403'}
2025-04-22 14:44:51,349 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:51,980 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:51,980 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2419'}
2025-04-22 14:44:51,980 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:52,068 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:52,068 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3795'}
2025-04-22 14:44:52,068 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:52,211 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:52,212 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:52,724 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:52,725 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '6174'}
2025-04-22 14:44:52,725 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:53,212 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:53,213 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:53,213 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:44:53,961 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:53,961 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8327'}
2025-04-22 14:44:53,962 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:54,213 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:54,213 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:55,024 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:55,024 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3603'}
2025-04-22 14:44:55,024 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:55,213 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:55,214 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:56,214 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:56,214 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:56,684 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:56,684 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '403'}
2025-04-22 14:44:56,684 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:56,873 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:56,873 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2419'}
2025-04-22 14:44:56,873 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:57,215 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:57,216 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:57,220 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:57,220 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3795'}
2025-04-22 14:44:57,220 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:57,959 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:57,959 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '6174'}
2025-04-22 14:44:57,959 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:58,216 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:58,216 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:44:58,216 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:44:59,165 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:44:59,165 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8327'}
2025-04-22 14:44:59,165 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:44:59,217 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:44:59,218 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:00,219 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:00,220 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:01,220 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:01,221 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:02,222 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:02,222 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:03,222 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:03,223 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:03,223 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:04,224 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:04,225 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:05,225 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:05,226 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:06,105 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:06,106 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3603'}
2025-04-22 14:45:06,106 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:06,227 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:06,228 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:06,479 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:06,479 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3795'}
2025-04-22 14:45:06,479 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:06,547 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:06,547 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2419'}
2025-04-22 14:45:06,547 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:06,684 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:06,684 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '403'}
2025-04-22 14:45:06,684 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:07,229 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:07,230 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:07,872 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:07,872 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '6174'}
2025-04-22 14:45:07,872 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:08,094 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:08,094 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8327'}
2025-04-22 14:45:08,094 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:08,230 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:08,230 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:08,230 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:09,231 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:09,231 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:10,232 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:10,233 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:11,233 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:11,233 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:12,234 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:12,235 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:13,235 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:13,236 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:13,236 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:14,238 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:14,238 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:15,239 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:15,241 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:16,241 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:16,242 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:17,242 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:17,243 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:18,244 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:18,245 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:18,245 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:19,246 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:19,247 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:20,248 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:20,249 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:21,249 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:21,250 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:22,251 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:22,252 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:23,252 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:23,252 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:23,252 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:23,572 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:23,572 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3603'}
2025-04-22 14:45:23,572 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:24,253 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:24,254 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:24,513 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:24,514 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2419'}
2025-04-22 14:45:24,514 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:24,623 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:24,623 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '403'}
2025-04-22 14:45:24,623 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:25,255 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:25,256 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:25,641 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:25,641 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3795'}
2025-04-22 14:45:25,641 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:25,649 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:25,650 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8327'}
2025-04-22 14:45:25,650 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:26,256 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:26,256 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:27,005 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:27,005 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '6174'}
2025-04-22 14:45:27,005 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:27,258 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:27,259 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:28,260 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:28,260 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:28,261 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:29,260 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:29,260 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:30,261 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:30,261 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:31,262 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:31,263 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:32,264 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:32,265 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:33,264 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:33,265 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:33,265 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:34,266 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:34,266 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:35,267 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:35,268 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:36,269 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:36,269 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:37,270 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:37,271 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:38,272 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:38,272 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:38,272 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:39,273 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:39,274 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:40,275 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:40,275 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:41,275 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:41,275 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:42,277 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:42,278 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:43,278 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:43,279 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:43,280 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:44,280 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:44,281 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:45,281 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:45,281 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:46,282 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:46,283 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:47,284 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:47,285 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:48,286 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:48,287 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:48,288 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:49,288 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:49,289 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:50,290 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:50,291 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:51,292 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:51,292 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:52,293 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:52,294 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:53,295 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:53,296 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:53,296 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:54,297 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:54,298 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:55,298 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:55,299 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:56,301 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:56,302 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:56,414 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-metadata.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064441Z&X-Goog-Expires=86399&X-Goog-Signature=063763e2324ff876ad4fe748285a4f7fec0b3f3411752d542ea1d90e29ac974a4f2f60f9d71720ba11ce2278327a2b696c8f0825fe777497b609ccf5d2f570d01151730afe35c14cbe39c76bb037c756ec809a29838a12f081f16929455f9bd3e034c32bf1e1fabc7aba82149c0a1771306e6e7357b15facb9f60f6aae5ed1d704c917c4529d14ea8cedbc651bf3acb6eaa0d0615d1a16fb0944d5ae4a3fbca69d3f9a6491cedaa4691df2e10dd1a7cc7d4790c4c2df1229c603dfbc8eb0b21280dd39a803cfcf041ad9284d8eaaf8538adfa36425820025739dd6e66581143117287842baa806ef8c8e541d94cb0e105dd745271ad10843fd572a8b2f940ebd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:56,414 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3603'}
2025-04-22 14:45:56,415 ERROR   wandb-upload_0:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:57,303 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:57,304 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:58,305 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:58,306 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:58,307 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:45:58,338 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/requirements.txt?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a55584eb79c4008726740894f16ebca8a1cfd90c45e02bab28419c67bae301404317e6d2ac390a94fc96ee7870e5cfeab0bdbad753d625b217a71f30a482bff75d05661d84002b4d3e69e44fcf60bfa17ccad86c38bd08c83cd7b156138ba4420730e86b42dad2e9574a257678ba9c0f7025931a3d6b162d3c2bd06eda14f77bb0c5d64e8f9a89c13176670ef74339aa236e877fc3301241e6ea0026460aaa1b153f69af1dfb2f8a8ae99fdcc54e42067704f1a9cf1f1807e7be5db1f581c1d216547c0aba59c4f8213d1afc0c9c3ba33fb1a39a2150203bcec7a41c717159756dc5f797a7802261f5876beeef6ea2d5741657d541b34c5e0cf9c22784a21632&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:58,339 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2419'}
2025-04-22 14:45:58,339 ERROR   wandb-upload_5:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:59,306 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:45:59,307 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:45:59,729 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/wandb-summary.json?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1093d36a90445a7b175cea0cd2fe7ca73d2442a3733466a5ab9693186382404494ebd246cfb910703db8ebcbdd601fccd51572b17af813f96dca86afc2ebd04d715b2e19f8b410e527febd5477d21f0e5ff8f40913666e399c101b14862995792c3de8dad05df0b5453f573fba622c173560fa6b6f2fe5440a07594038f8aa586afe4926e3cfeb7546f73c859857618443a300659348c928b1b7c610cee9d643ae74ae4cc4bd20e62fc7a3fb4e7095947f83f8c6f1a906804a4e1bc8e1eebf630029d3b7a39b244ae811de24d5e1a965c422b087596dcc898d8b664a171e89bd436b5d8936962fc869b76b823a904fef383068d902aa4dc333102e6edf9a5cbd&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:59,729 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '403'}
2025-04-22 14:45:59,729 ERROR   wandb-upload_2:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:45:59,730 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/output.log?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064448Z&X-Goog-Expires=86399&X-Goog-Signature=44f17fddede3f04950ff03ba7c15ace82fef17d680f2bf99c55d813a4d7f4676432346eac61b1c191067a92ff382edc485245b00299b591d8e361fef423974a76a363d30e44184303e1424a1b6c2fd505e671f0e1668963654f826e9504daec477caf3743174833ae3265c5f7e360ad5836162ba22be75814e31fc043e0435b38c7224bfddf2ebbd7dcdf12f56c8680fd6b7a4492b433569e81739168eb72477402388a63c5948e9fd4196556a33eb0e1a87635cf7cb57e45f1422e2ffc1fbfdb131ce471adef13624a98a2eaff112d44e6c7c0faed2508a69fb5649a002911953aa5198b7334ec48934d82188f0c0368611a5eaf02d7284a72c79fe1586f94a&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:45:59,730 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8327'}
2025-04-22 14:45:59,730 ERROR   wandb-upload_3:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:46:00,309 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:00,310 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:01,309 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:01,310 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:01,420 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/config.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=1afa073a357953b3213abd770c0d224c997c6d48855b3cd0dbbdc07858e6af123c632fd5086671a8b838046b7255ef9856cd57e65f3eda5f766b9f282f4bafbf7e385ad6f5be2b355ccd3af6bf2fcacab83a993787fc8edbe373980579d13fdb503a04fdf045f40ee7c3407e39bcfde4f9df97c17ef27f3b9d4ab4777dd33d03bba5e3faa9d89c5b0e1d11faacb39aaafdb53661bb6ed021880cc8636878c5c760ba49125079a5bfbeb8c84586aa6ca036a54e45458cf107d720719454d7bb07ff54814fe4d65eb78b40367a839948249059467ededc03e503756b3b7d9b1da0310f697be69c9ea63b000108e824cf731d0ff48ad90f83a52b7337b3b3ad14b5&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:46:01,421 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3795'}
2025-04-22 14:46:01,421 ERROR   wandb-upload_4:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:46:02,312 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:02,312 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:03,312 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:03,312 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:03,312 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:46:04,313 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:04,314 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:05,313 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:05,313 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:05,432 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2887] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/OA-MBRL/moore-tdmpc2/6wyz8vcn/conda-environment.yaml?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com%2F20250422%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250422T064446Z&X-Goog-Expires=86399&X-Goog-Signature=a8a50f77e8584c69f3ef954eb6d101f68f5f976a6ec28ecb381b976b43a9d89b630da200a0ce5802e26c40a54a96682996637ff857084ae9b223029779e1bed78bff80a46ff65b992ae113cd4d52852349aa1bf51b1d1c9259f935b22434e0910f3b9d87796a0f48fe3e8d239e47e959e69471ee4c45085f0f185b8c23a96aa1d273a95f1db14acd2587e447e199d3666de0bb5294a168388363308b431d4ed212526bd6d8a6048acfa86360b69ef2fb7be0e108b4b7459904f795596d30b088e6ea1d030c4c01ce3721ef993078cfffb805cbb93aaebd83fc5a5e30215796aa294f462ce8a3b9df01c00ac4de334985efd67fa2bb48b5c110c5b79d9f0870c8&X-Goog-SignedHeaders=host&X-User=levi1024 (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1147)')))
2025-04-22 14:46:05,433 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2889] upload_file request headers: {'User-Agent': 'python-requests/2.32.3', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '6174'}
2025-04-22 14:46:05,433 ERROR   wandb-upload_1:578817 [internal_api.py:upload_file():2891] upload_file response body: ''
2025-04-22 14:46:06,314 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:06,315 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:07,315 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:07,316 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:08,316 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:08,316 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:08,316 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:46:09,317 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:09,318 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:10,318 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:10,319 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:11,318 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:11,319 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:12,319 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:12,319 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:13,320 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:13,321 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 14:46:13,322 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:13,650 WARNING StreamThr :578817 [internal.py:is_dead():413] Internal process exiting, parent pid 578703 disappeared
2025-04-22 14:46:13,651 ERROR   StreamThr :578817 [internal.py:wandb_internal():151] Internal process shutdown.
2025-04-22 14:46:14,321 DEBUG   HandlerThread:578817 [handler.py:handle_request():158] handle_request: poll_exit
2025-04-22 14:46:14,321 INFO    HandlerThread:578817 [handler.py:finish():882] shutting down handler
2025-04-22 14:46:14,322 INFO    WriterThread:578817 [datastore.py:close():296] close: /home/levi/Documents/MOORE-TDMPC/moore_tdmpc/logs/mw-reach/42/moore-tdmpc2-mt30/wandb/run-20250422_144436-6wyz8vcn/run-6wyz8vcn.wandb
2025-04-22 14:46:14,322 DEBUG   SenderThread:578817 [sender.py:send_request():406] send_request: poll_exit
2025-04-22 14:46:14,322 INFO    SenderThread:578817 [sender.py:finish():1608] shutting down sender
2025-04-22 14:46:14,322 INFO    SenderThread:578817 [file_pusher.py:finish():169] shutting down file pusher
2025-04-22 14:46:14,322 INFO    SenderThread:578817 [file_pusher.py:join():175] waiting for file pusher
